nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-parameter-contexts.js

```diff
@@ -2397,8 +2397,6 @@
          * @param parameterToSelect   Optional, name of the parameter to select in the table.
          */
         showParameterContext: function (id, readOnly, parameterToSelect) {
-            parameterCount = 0;
```

Comment:
This is needed as the count gets reset every time the `#parameter-context-dialog` is opened and this count is used to track the index of any new parameters. Instead of removing this line try calling `resetUsage()` before we reset the `parameterCount = 0;`